rest/data/project_test.go

```diff
@@ -442,6 +442,20 @@ func (s *ProjectConnectorGetSuite) TestUpdateProjectVars() {
 	s.NoError(s.ctx.UpdateProjectVars("not-an-id", &newVars, false))
 }
 
+func (s *ProjectConnectorGetSuite) TestUpdateProjectVarsByValue() {
```

Comment:
Can we maybe add a test where the vars are nested?

Reply:
same comment as before (no nested vars)